File: ProAgil.Api/Model/Evento.cs
namespace ProAgil.Api.Model {
    public class Evento {
        public Evento (int eventoId, string local, string dataEvento, string tema, int qtdePessoas, string lote, string imagemUrl) {
            this.EventoId = eventoId;
            this.Local = local;
            this.DataEvento = dataEvento;
            this.Tema = tema;
            this.QtdePessoas = qtdePessoas;
            this.Lote = lote;
            this.ImagemUrl = imagemUrl;

        }
        public int EventoId { get; set; }
        public string Local { get; set; }
        public string DataEvento { get; set; }
        public string Tema { get; set; }
        public int QtdePessoas { get; set; }
        public string Lote { get; set; }
        public string ImagemUrl { get; set; }
    }

}

File: ProAgil-App/src/app/_models/Evento.ts
import { Lote } from './Lote';
import { RedeSocial } from './RedeSocial';
import { Palestrante } from './Palestrante';

export class Evento {
    constructor( ) { }
id: number;
local: string;
lotes: Lote[];
dataEvento: Date;
tema: string;
qtdePessoas: number;
imagemUrl: string;
telefone: string;
email: string;
list: Lote[];
redeSociais: RedeSocial[];
palestranteEventos: Palestrante[];

}
